<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreeDDialog</class>
 <widget class="QDialog" name="ThreeDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetThreeD" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDisplay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>300</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Display</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QCheckBox" name="chkOXY">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>139</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Grid OXY</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkOYZ">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>177</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Grid OYZ</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkAxis">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>120</y>
        <width>59</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Axis</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkOXZ">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>158</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Grid OXZ</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="grpView">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>16</y>
        <width>125</width>
        <height>40</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rbViewShading">
         <property name="text">
          <string>View Shading</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbViewMesh">
         <property name="text">
          <string>View Mesh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpType">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>196</y>
        <width>69</width>
        <height>40</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="rbShape">
         <property name="text">
          <string>shape</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbColor">
         <property name="text">
          <string>color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
